release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -419,6 +419,26 @@ The Network Observability Operator releases updates independently from the {prod
 //[id="ocp-4-15-cluster-cloud-controller-manager-operator"]
 //=== Cloud controller managers for additional cloud providers
 
+[discrete]
+[id="ocp-4-15-planned-psa-restricted-enforcement"]
+=== Future restricted enforcement for pod security admission
+
+Currently, pod security violations are shown as warnings and logged in the audit logs, but do not cause the pod to be rejected.
+
+Global restricted enforcement for pod security admission is currently planned for the next minor release of {product-title}. When this restricted enforcement is enabled, pods with pod security violations will be rejected.
```

Comment:
In general, I thought we weren't supposed to mention features of future releases in the docs[1]. I assume there's a justifiable reason for including this info in the docs, though.

[1] https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future

Reply:
Yes, this one is okay because it is a warning to prepare. It has the proper fluffy warning that allows for flexibility if the plans change (which they have done, several times). Thanks for checking!